requirements.txt

```diff
@@ -5,7 +5,7 @@ google-cloud-pubsub
 pyarrow
 pyyaml >= 5.1
 sqlalchemy
-lsst-dax-apdb @ git+https://github.com/lsst/dax_apdb@main
+lsst-dax-apdb @ git+https://github.com/lsst/dax_apdb@tickets/DM-52215
```

Comment:
I don't think you want to commit this

Reply:
> I don't think you want to commit this

Indeed, this is only needed for tests in Github Action. I'll remove it once dax_apdb is merged.